invenio.cfg

```diff
@@ -336,8 +353,16 @@ RDM_CUSTOM_FIELDS = [
     KeywordCF(name='openbiodiv:TaxonomicConceptLabel', multiple=True),
     # obo
     RelationshipListCF(name='obo:RO_0002453'),
+    # meeting
+    *MEETING_CUSTOM_FIELDS,
 ]
 
+# Custom fields UI components
+RDM_CUSTOM_FIELDS_UI = [
+    MEETING_CUSTOM_FIELDS_UI
```

Comment:
Should we make consistent use of lists. In this case meeting only has one section, however, what if they were more? then we end up with a mix of `*` and non `*`. I see two options:
a) Enforce one section per `XYZ_CUSTOM_FIELDS_UI` in which case i'd rename to `MEETING_CUSTOM_FIELDS_UI_SECTION`
b) Make `MEETING_CUSTOM_FIELDS_UI` a list then `*` here.

Reply:
I see your point and I do agree, right now we can only have one section per custom field. However, and correct me if I am wrong, that behavior is part of the custom fields interface (e.g. `XYZ_CUSTOM_FIELDS_UI` is an object with  a property `section` and `fields`). 

For `RDM_CUSTOM_FIELDS_UI`, it will look like this:

```python
RDM_CUSTOM_FIELDS_UI = [
    MEETING_CUSTOM_FIELDS_UI,
    JOURNAL_CUSTOM_FIELDS_UI,
    ...
]
```

Reply:
My question is not on any inner behaviours, it focuses on naming/user experience. If I have:
- RDM_CUSTOM_FIELDS_UI
- MEETING_CUSTOM_FIELDS_UI
- JOURNAL_CUSTOM_FIELDS_UI
- XYZ_CUSTOM_FIELDS_UI

It gives me the impresion that they are analog, meaning they can be replaced. For example:

```python
RDM_CUSTOM_FIELDS_UI  = XYZ_CUSTOM_FIELDS_UI
```

When in reality is not true, because the last 3 of the list above are "parts" of the first one. That's why I wanted to suffix with `SECTION`. However, the name becomes long.